docs: create metadata

jducnuigeen opened the pull request:
Fixes: #[243](https://github.com/linz/qgis-buildingsgeneralisation-plugin/issues/243)
Made to be as concise as possible, with the idea the Data Dictionary is more detailed.
Check attribution in the section "Metadata Constraints" since this has more than one attributing source.
Metadata File identifier created from ANZMetLite.

pkinglinz committed:
feat: insert sql

sql required to copy data from a temp table used into the facilities table. The temp table stores the data imported from the gpkg in qgis. Upted to use drop table as easier to understand than altenrative, and to create and populate table rather than use create table as so primary key correctly identified

pkinglinz committed:
fix: remove insert scripts

insert only sued once so removing from repo
fix: remove inserts again

jducnuigeen committed:
docs: create metadata

jducnuigeen committed:
docs: create metadata

pkinglinz committed:
Merge pull request #3 from linz/insert_facilities_script

feat: insert sql

pkinglinz committed:
feat: copy paste from nz_buildings

Have not adjusted rst files, but adjusted other files to reference Facilities instead of buildings

jducnuigeen committed:
docs: update on metadata

jducnuigeen committed:
docs: update on metadata

pkinglinz committed:
feat: produces published_data html

Doesn't populate the descriptions on this table

dwsilk commented on metadata/nz_facilities.xml:
```diff
@@ -0,0 +1,364 @@
+
+A more detailed description of NZ Facilities can be found in the NZ Facilities Data Dictionary [LINK]. This Data Dictionary also includes information on how NZ Facilities were used to support the attribution of NZ Building Outlines.
+
+NZ Facilities contains data sourced from NationalMap, Ministry of Education and Ministry of Health licensed for reuse under CC BY 4.0.
```
```markdown
NZ Facilities contains data sourced from [NationalMap](https://www.nationalmap.co.nz/), [Ministry of Education](https://www.educationcounts.govt.nz/) and [Ministry of Health](https://www.health.govt.nz/) licensed for reuse under [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/).
```

dwsilk replied:
Should contain links ^

dwsilk commented on metadata/nz_facilities.xml:
```diff
@@ -0,0 +1,364 @@
+
+NZ Facilities contains data sourced from NationalMap, Ministry of Education and Ministry of Health licensed for reuse under CC BY 4.0.
+
+**Related NZ Facilities data**
```
Just `**Related data**` ?

dwsilk commented on metadata/nz_facilities.xml:
```diff
@@ -0,0 +1,364 @@
+          <gmd:statement>
+            <gco:CharacterString>
+**Source**
+NZ Facilities contains data sourced from NationalMap, Ministry of Education and Ministry of Health licensed for reuse under CC BY 4.0.
```
Same as above.. add links.

dwsilk commented on metadata/nz_facilities.xml:
```diff
@@ -0,0 +1,364 @@
+
+**Attributes**
+• The FACILITY_ID provides a unique id for each facility.
+• The NAME attribute provides a facility name which has consistent naming convention applied. This name may directly match the name provide by the authoritative source.
```
provided (fix typo)

dwsilk commented on metadata/nz_facilities.xml:
```diff
@@ -0,0 +1,364 @@
+**Attributes**
+• The FACILITY_ID provides a unique id for each facility.
+• The NAME attribute provides a facility name which has consistent naming convention applied. This name may directly match the name provide by the authoritative source.
+• The EXTERNAL_NAME attribute provides the facility name used by the authoritative source.
```
maybe this should be `source_name` ? probably too late to suggest that and I know that `external` matches the terminology we have used elsewhere..

jducnuigeen committed:
docs: just test of DD

jducnuigeen committed:
docs: update requirements

jducnuigeen committed:
Merge pull request #5 from linz/data_dictionary

Data dictionary

jducnuigeen committed:
docs: update setup

jducnuigeen committed:
Merge pull request #6 from linz/docs_updates

docs: update setup

jducnuigeen committed:
docs: metadata updates

jducnuigeen committed:
docs: metadata updates

jducnuigeen committed:
fix: updated db column names

Douglas-K committed:
docs: metadata updates

Douglas-K committed:
docs: metadata updates

jducnuigeen committed:
fix: updated tables

jducnuigeen committed:
Merge pull request #7 from linz/db_column_update

fix: updated db column names

pkinglinz committed:
feat: svg of diagram

having diagram in repo lets anyone adjust in the future

pkinglinz committed:
Merge pull request #8 from linz/facilities_diagram

feat: svg of diagram

pkinglinz committed:
feat: update image references

and removed old images

pkinglinz committed:
feat: description

jducnuigeen committed:
docs: fix links

jducnuigeen committed:
docs: link fixing

jducnuigeen committed:
fix: link update

pkinglinz committed:
feat: updated attribute descriptions

jducnuigeen committed:
docs: fix sections

pkinglinz committed:
feat: overvew diagram

Douglas-K committed:
docs: metadata updates

Douglas-K committed:
docs: metadata updates

pkinglinz committed:
feat: attribute description

and extended underlines to remove error

Douglas-K committed:
docs: metadata updates

Douglas-K committed:
docs: metadata updates

dwsilk committed:
docs: grammar/spelling/restructured text clean up

jducnuigeen committed:
docs: minor updates after DS and DK comments

Douglas-K committed:
docs: metadata updates

Douglas-K committed:
docs: metadata updates

jducnuigeen committed:
docs: testing tables setup

jducnuigeen committed:
docs: update background

jducnuigeen committed:
docs: update tables

jducnuigeen committed:
docs: update overview

jducnuigeen committed:
docs: table fix

jducnuigeen committed:
docs: fix 2 tables

jducnuigeen committed:
docs: fix tables 3

jducnuigeen committed:
docs: fix tables 4

jducnuigeen committed:
docs: update svg

jducnuigeen committed:
docs: update backwards table item

jducnuigeen committed:
docs: update use table

Douglas-K committed:
docs: metadata updates

Douglas-K committed:
docs: metadata updates

jducnuigeen committed:
docs: updates to tables and conf file

Douglas-K committed:
docs: metadata updates

Douglas-K committed:
docs: metadata updates

jducnuigeen committed:
docs: update logo path and URL

pkinglinz committed:
docs: diagrams

updated diagrams particularly the overview.

updated sql for the lds table so id no longer serial but an integer as will be a copy form main table

pkinglinz committed:
Merge remote-tracking branch 'origin/data_dictionary_updates' into data_dictionary_updates

jducnuigeen committed:
docs: add verifying image

jducnuigeen committed:
docs: update image captions

jducnuigeen committed:
docs: image caption corrections

jducnuigeen committed:
docs: update URL

jducnuigeen committed:
Merge pull request #9 from linz/data_dictionary_updates

Updated Data Dictionary

Douglas-K committed:
Merge remote-tracking branch 'origin/metadata' into metadata
